### DISCUSSION TITLE
Why are the default figures monospaced?

### DISCUSSION THREAD

**Discussion by Arno-Enslin** [adobe-fonts/source-serif#135] [Q&A] [answered]
Why are the default figures monospaced, but not lining only?

**Comment by adrientetar** [comment]
I think mostly for legacy reasons that monospaced figures let you typeset a spreadsheet in an application that doesn't support OTL.

**Comment by Arno-Enslin** [comment]
I thought about changing that for a webfont version of Source Serif. But probably it is as broad as it is long.

**Comment by frankrolf** [comment]
This is a convention. Most of Adobe’s typefaces intended for body text (and also many other foundries’ fonts) follow it.

> A webfont version of Source Serif

— You are aware of the web fonts available here I assume:
https://github.com/adobe-fonts/source-serif-pro/releases/latest

**Comment by frankrolf** [comment, marked as answer]
Most convincing explanation I’ve found (by @mbutterick):

> No ver­sion of Mi­crosoft Ex­cel sup­ports Open­Type fea­tures. So if you want tab­u­lar fig­ures in your Ex­cel spread­sheet—and I think you do—you must limit your­self to fonts with tab­u­lar fig­ures in the de­fault fig­ure po­si­tions.

From: http://practicaltypography.com/alternate-figures.html

**Comment by Arno-Enslin** [comment]
@ frankrolf

> You are aware of the web fonts available here I assume

Yes, but I plan to build subsets. As far as I have seen, the FontTools subset script works very fine. I want to keep the file sizes of the fonts as small as possible. But because the tabular lining and the normal lining figures have the same outlines, I am probably keeping both of them.

By the way: Source Serif looks great on screen.

**Comment by mbutterick** [comment]
FWIW I don’t follow this convention for my own fonts. For my customers, it makes more sense for the default figures to be proportional. I generate a separate set of fonts (using Python) with the tabular figures in the default positions. That way, the minority of users who want tab figures as the default (e.g., MS Excel users) don’t need to inconvenience the majority who want the opposite.

Still, for an open-source font, I think @frankrolf's choice is wise. For those unfamiliar with these details, tabular figures obey the principle of least astonishment.
